### ISSUE TITLE
.Net 4.6.1 referencing issues

### ISSUE THREAD

**Issue by zoharshavit** [microsoftgraph/msgraph-sdk-dotnet#1493]
Hi Team,
I was tasked to write a C# component to add, update and delete users from Azure AD.
After a bit research, I came to the conclusion that Graph is the way to go.
I am currently working with .Net 4.6.1 (currently unable to upgrade), so I found Graph's latest version supporting this .Net version - 3.35.0.
After installing and trying to run, I'm getting an error saying Newtonsoft.Json 6 was not found.
I'm currently working with Newtonsoft.Json 9, and Graph's version 3.35 dependency list is saying the it depends on Graph.Core >= 1.25.1.
Looking at Graph.Core 1.25.1 dependency list I see that when using .Net 4.5 (and above I presume...) it depends on Newtonsoft.Json **>=** 6.0.1.
So, I expected it to work with my Newtonsoft.Json 9.
When it didn't, I looked for a more current version of Graph.Core, thinking maybe it will support a more current Newtonsoft.Json version, but as soon as I look into version 2.0.0 I see that it only support .Net 4.6.2 and above...

Is there a working configuration when woking with .Net 4.6.1 and Newtonsoft.Json 9.0.1?
![Graph](https://user-images.githubusercontent.com/26147773/191500538-4087c558-d063-4a39-b7e8-bce923ae7b0f.png)
![Graph Core](https://user-images.githubusercontent.com/26147773/191500542-aa4a7548-4428-4fa2-9f9e-3fa38cf05bfc.png)

**Comment by andrueastman** [comment]
Thanks for raising this @zoharshavit,

I believe it should be possible to use a higher version of the Newtonsoft.Json library. Any chance you could try removing installing the Newtonsoft.Json first then add the Microsoft.Graph dependency? 

https://github.com/microsoftgraph/msgraph-sdk-dotnet/tree/support/3.x#newtonsoftjson-version

If not, could you share a sample project with the error to help us understand the issue better?

**Comment by zoharshavit** [comment]
Hi @andrueastman ,
Thanks for your reply.
I actually saw this comment about installing Newtonsoft.Json first and tried it.
When installing Graph after installing Newtonsoft.Json I saw a difference where it did not install Newtonsoft.Json again, which made me feel hopeful, but to my disappointment - it did not make any difference to the runtime error I got....
In my attempt to restore the issue so that I could upload a working (or not working really) here, I created a new project from scratch, and installed all components.
Problem was - It worked perfectly... so I really don't know what is the problem, but I guess its something in my env.
For now, I switched my approach, and currently am not using Graph but making the calls directly using HttpRequests, which is exactly what I was hoping to avoid when choosing Graph.
Maybe after upgrading to .Net 4.8 I'll have another go at it.
Thanks again for trying to help.
